README.md

```diff
@@ -68,7 +68,7 @@ As a last resort, you can manually set each device to use Pi-hole as their DNS s
 
 ## Pi-hole is free but powered by your support
 
-There are many reoccurring costs involved with maintaining free, open source, and privacy-respecting software; expenses which [our volunteer developers](https://github.com/orgs/pi-hole/people) pitch in to cover out-of-pocket. This is just one example of how strongly we feel about our software and the importance of keeping it maintained.
+There are many reoccurring costs involved with maintaining free, open-source, and privacy-respecting software; expenses which [our volunteer developers](https://github.com/orgs/pi-hole/people) pitch in to cover out-of-pocket. This is just one example of how strongly we feel about our software and the importance of keeping it maintained.
```

Comment:
I guess this is the correct usage in this case?

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source

Reply:
As noted in the guide you've linked, "Hyphenate open-source as an adjective preceding a noun, as in open-source software". So I think in this case "open source" is missing a hyphen since it is an adjective preceding "software" (noun).